### PR TITLE
update instructions for install of my-app

### PR DESCRIPTION
**Pull request by Coding-with-Adam**
This will ensure alignment with the rest of the tutorial.

### CODE REVIEW

**Comment by ljcutts** [comment]
Hello @Coding-with-Adam, I don't agree with this PR because this will cause users to use typescript and not everyone uses Typescript. Along with TailwindCSS.

Captura de pantalla 2023-05-08 a la(s) 10 40 19 a m

This would be the ideal options. Edit your PR similar to this and I will merge it.

File: NFT-Collection.md [hunk 1]
```diff
@@ -345,7 +345,7 @@ To create this `my-app`, in the terminal point to NFT-Collection folder and type
   npx create-next-app@latest
 ```
 
-and press `enter` for all the questions
+and press `enter` for all the questions, except for the question on "Use App Router (recommended)?". Select 'No' when asked.
```
Comment (Coding-with-Adam) [inline NFT-Collection.md:348]:
```suggestion
You will be prompted with questions. Follow the image below to answer the questions accordingly. 

![image](https://user-images.githubusercontent.com/32049495/236994121-bdea21fe-3afc-4c33-b736-806585d5df34.png)

.
```

**Comment by Coding-with-Adam** [comment]
hey @ljcutts 
I modified the instructions. Let me know if the image I upload worked. I can't see whether it uploaded correctly.

**Comment by ljcutts** [comment]
Thank you @Coding-with-Adam, this works.

**Comment by ljcutts** [comment]
@Coding-with-Adam, can you do this for the other tutorials as well within the Sophomore and Junior Track?

**Pull request merged**